dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -70,13 +70,25 @@
       <Priority>Undefined</Priority>
       <IsPublic>true</IsPublic>
       <IncludeInWindowsAppx>true</IncludeInWindowsAppx>
+      <!-- Valid value: Latest, V2dot5 and Any, default is Any -->
+      <UseVisualStyle>Any</UseVisualStyle>
     </Page>
   </ItemDefinitionGroup>
   <ItemDefinitionGroup>
     <PRIResource>
       <IsPreviewResource>False</IsPreviewResource>
     </PRIResource>
   </ItemDefinitionGroup>
+  <ItemGroup>
+    <MergeXamlConfig Include="LatestVisualPages">
+      <FilterAttribute>UseLatestStyle</FilterAttribute>
+      <Postfix />
+    </MergeXamlConfig>
+    <MergeXamlConfig Include="V2dot5VisualPages">
+      <FilterAttribute>UseV2dot5Style</FilterAttribute>
+      <Postfix>_2dot5</Postfix>
```

Comment:
In the XAML file names you have `_v2.5` as the postfix, whereas here you have `_2dot5`.  We should make it consistent which we use.

Reply:
actually it is matching with the enum winrt::StylesVersion::WinUI_2dot5 because enum doesn't support the '.'.
 If you think 2.5 is more reasonable for file name, I can make the change.

Reply:
I'm mainly thinking just that we should keep filenames consistent.  I would personally prefer _v2.5.